gef-ui/server/controllers/amendments/cover-end-date/validation.test.ts

```diff
@@ -0,0 +1,92 @@
+import { applyStandardValidationAndParseDateInput, COVER_END_DATE_MAXIMUM_YEARS_IN_FUTURE, DayMonthYearInput } from '@ukef/dtfs2-common';
+import { add, format, startOfDay, getUnixTime } from 'date-fns';
+import { validateAndParseCoverEndDate } from './validation';
+import { mapValidationError } from '../../../utils/map-validation-error';
+
+const valueName = 'cover end date';
+const valueRef = 'coverEndDate';
+
```

Comment:
I would suggest pick `now()` function and store the value in variable to save calling `new Date()` multiple timnes. Function is in `libs/common`.

```ts
const today = now();
```